### ISSUE TITLE
Plot export fails if no category field specified

### ISSUE THREAD

**Issue by misken** [misken/hillmaker#42]
If the optional category field is not specified, then the plots fail with:

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Cell In[11], line 1
----> 1 results_ex1 = hm.make_hills(scenario, stops_df, in_field, out_field,
      2                             start_date, end_date,
      3                             bin_size_minutes=bin_size_mins,
      4                             output_path=output_path,
      5                             export_dow_png=True,
      6                             export_week_png=True,
      7                             xlabel='xlabel',
      8                             ylabel='ylabel')

File ~\Anaconda3\envs\aap\Lib\site-packages\hillmaker\hills.py:252, in make_hills(scenario_name, stops_df, in_field, out_field, start_analysis_dt, end_analysis_dt, cat_field, bin_size_minutes, percentiles, cats_to_exclude, occ_weight_field, totals, cap, nonstationary_stats, stationary_stats, no_censored_departures, export_bydatetime_csv, export_summaries_csv, export_dow_png, export_week_png, xlabel, ylabel, output_path, edge_bins, verbosity)
    250 if export_week_png:
    251     with HillTimer() as t:
--> 252         for metric in summary_dfs['nonstationary']['dow_binofday']:
    253             fullwk_df = summary_dfs['nonstationary']['dow_binofday'][metric]
    254             fullwk_df = fullwk_df.reset_index()

KeyError: 'dow_binofday'
```

**Comment by misken** [comment]
An easy workaround is to create a dummy category field containing a single value. This bug is fixed in 0.4.6 which should be released later today.

**Comment by misken** [comment]
Bug fixed in commit d1739fd (v0.4.6).
